### PR TITLE
Feature/#23

### PR DESCRIPTION
**Pull request by shinriyo**
Detail for [Feature Request] Get only date or only time #23

### CODE REVIEW

**Comment by jogboms** [comment]
Hey @shinriyo 
Nice work, could you revert the `dartfmt` changes? or set your `lineWidth` to 120.

File: lib/src/extensions.dart [hunk 1]
```diff
@@ -31,13 +34,21 @@ extension DateTimeTimeExtension on DateTime {
   /// Subtracts the Duration from this DateTime returns the difference as a new DateTime object.
   DateTime operator -(Duration duration) => subtract(duration);
 
+  /// Returns only year, month and day
+  DateTime get date => DateTime(year, month, day);
+
+  /// Returns only the time
+  String get timeOfDay =>
```
Comment (jogboms) [inline lib/src/extensions.dart:41]: This should probably return a `Duration` instead.

File: lib/src/extensions.dart [hunk 2]
```diff
@@ -31,13 +34,21 @@ extension DateTimeTimeExtension on DateTime {
...
+      "${'$hour'.padLeft(2, '0')}:${'$minute'.padLeft(2, '0')}:${'$second'.padLeft(2, '0')}";
```
Comment (jogboms) [inline lib/src/extensions.dart:42]:
```suggestion
      hour.hours + minute.minutes + second.seconds;
```

**Comment by shinriyo** [comment]
> Hey @shinriyo
> Nice work, could you revert the `dartfmt` changes? or set your `lineWidth` to 120.
OK. I will set in the setting of AndroidStudio.
Settings->Editor -> Code style -> Dart -> Wrapping and braces (tab) -> changed Hard wrap to 120

File: lib/src/extensions.dart (hunk 2 above)
Comment (shinriyo) [reply lib/src/extensions.dart]: OK. I fix.

File: lib/src/extensions.dart (hunk 1 above)
Comment (shinriyo) [reply lib/src/extensions.dart]: Tjank you.

**Comment by shinriyo** [comment]
@jodinathan 
>  or set your lineWidth to 120.

Is `120` good practice or mentioned on the guideline of the Flutter team?
I want to add a rule for `analysis_options.yaml`.

**Comment by jodinathan** [comment]
> @jodinathan
> 
> > or set your lineWidth to 120.
> 
> Is `120` good practice or mentioned on the guideline of the Flutter team?
> I want to add a rule for `analysis_options.yaml`.

I think you mentioned the wrong user

**Comment by shinriyo** [comment]
@jodinathan 
> I think you mentioned the wrong user

I'm sorry.  Same j initial, so I mistook.

@jogboms 
I found it.
https://dart-lang.github.io/linter/lints/lines_longer_than_80_chars.html

**Comment by shinriyo** [comment]
@jogboms
Please review.

**Comment by jogboms** [comment]
Hey @shinriyo 

Sorry I got swamped with daily work.

**Comment by shinriyo** [comment]
@jogboms Thank you for merging.

**Pull request merged**